### DISCUSSION TITLE
problem with the new version on windows7 x64

### DISCUSSION THREAD

**Discussion by wojtal84** [cedricp/ddt4all#1074] [General]
Hello, after installing the last version, there is a problem with starting, error 0xc000007b appears

**Comment by Furtif** [comment]
> Hello, after installing the last version, there is a problem with starting, error 0xc000007b appears

https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170 get x64

**Comment by wojtal84** [comment]
unfortunately it didn't help
![20240413_132458](https://github.com/cedricp/ddt4all/assets/166631474/88561e92-2885-4092-a32b-169879aff1e8)

**Reply by Furtif** [reply]
https://github.com/cedricp/ddt4all/releases/download/v3.0.1/DTT4ALL-Windows-Installer-v3.0.1_Stable-x86-vcredist.exe you have test install this version ? Python 3.12 is not maybe ok for windows 7.

**Reply by Furtif** [reply]
Is my answer from before good?

**Comment by AndyIreland** [comment]
> https://github.com/cedricp/ddt4all/releases/download/v3.0.1/DTT4ALL-Windows-Installer-v3.0.1_Stable-x86-vcredist.exe you have test install this version ? Python 3.12 is not maybe ok for windows 7.

what positives / additives / changes does this 3.12 version bring over python27 version?

**Reply by Furtif** [reply]
https://github.com/cedricp/ddt4all/discussions/1087#discussioncomment-9280567

**Reply by KarelSvo** [reply]
https://github.com/cedricp/ddt4all/discussions/1087#discussioncomment-9282503

Win 7-11 does not require Python 3, also works with Python 2 https://github.com/cedricp/ddt4all/discussions/1088#discussioncomment-9286930

Python 3 is necessary for current Linux operating systems. Because Linux does not work without Python and Python 2 and PyQT4 are difficult or impossible to install in current Linux versions.

This does not change if you remove the linked topic from the current list of discussions by closing it.

I am not against Python 3, I use the Python 3 PyRen version on Linux . Manufacturer CLIP diagnosis and ddt2000 programming all inclusive.

**Comment by Furtif** [comment]
> > https://github.com/cedricp/ddt4all/releases/download/v3.0.1/DTT4ALL-Windows-Installer-v3.0.1_Stable-x86-vcredist.exe you have test install this version ? Python 3.12 is not maybe ok for windows 7.
> 
> what positives / additives / changes does this 3.12 version bring over python27 version?

win 7 not need 2,7 py is ok of 3,x

**Reply by Furtif** [reply]
deps platform or bad help

**Reply by Furtif** [reply]
on the other hand, the x86 version works even on win7 x64, and it won't be necessary to use py 2, just be alive, and follow what's going on

**Reply by Furtif** [reply]
for info if your system KO or formated rapport you to https://legacyupdate.net 
and fine
